spring-boot-project/spring-boot-devtools/src/main/java/org/springframework/boot/devtools/livereload/Connection.java

```diff
@@ -77,7 +77,7 @@
 	 * @throws Exception in case of errors
 	 */
 	void run() throws Exception {
-		if (this.header.contains("Upgrade: websocket") && this.header.contains("Sec-WebSocket-Version: 13")) {
+		if (this.header.contains("Upgrade: websocket") && this.header.toLowerCase().contains("sec-websocket-version: 13")) {
```

Comment:
Maybe we can maintain the minimum changes, so here we can go back to the original state.

Reply:
I'm not sure I understand. Do you mean to do `"Sec-WebSocket-Version: 13".toLowerCase()` instead?

Reply:
I mean revert this line. It seems superfluous to me, doesn't it?

Reply:
I don't think so. Proxies may pass the header with the casing `Sec-Websocket-Version` (lowercase `s` instead of uppercase `S`) so this comparison would fail.

Reply:
I agree with what you said, I get it.